ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/BindingVerifier.kt

```diff
@@ -17,10 +17,8 @@ import com.jayway.restassured.response.Response
 import org.codice.compliance.SAMLBindings_3_4_6_a
 import org.codice.compliance.SAMLComplianceException
 import org.codice.compliance.utils.TestCommon.Companion.IDP_ERROR_RESPONSE_REMINDER_MESSAGE
-import org.codice.compliance.utils.determineBinding
-import org.codice.security.saml.SamlProtocol
 
-abstract class BindingVerifier(open val response: Response) {
+abstract class BindingVerifier(open val httpResponse: Response) {
```

Comment:
❓ Is there a reason `httpResponse` is open? It looks like it can be a closed value that the children access.

Reply:
Fixed